utilities/rate_limiters/write_amp_based_rate_limiter.cc

```diff
@@ -297,46 +308,41 @@ Status WriteAmpBasedRateLimiter::Tune() {
 
   int64_t prev_bytes_per_sec = GetBytesPerSecond();
 
-  bytes_sampler_.AddSample(duration_bytes_through_ * 1000 / duration_ms);
-  highpri_bytes_sampler_.AddSample(duration_highpri_bytes_through_ * 1000 /
-                                   duration_ms);
-  if (bytes_sampler_.AtTimePoint()) {
-    long_term_bytes_sampler_.AddSample(bytes_sampler_.GetFullValue());
-    long_term_highpri_bytes_sampler_.AddSample(
-        highpri_bytes_sampler_.GetFullValue());
+  // Loop through the actual time slice to make sure bytes flow from long period
+  // of time is properly estimated.
```

Comment:
```suggestion
  // of time is properly estimated when the compaction rate is low.
```
This helps me understand the motivation more easily.